deep_qa/layers/subtract_minimum.py

```diff
@@ -0,0 +1,45 @@
+from keras import backend as K
+
+from deep_qa.layers.masked_layer import MaskedLayer
+
+class SubtractMinimum(MaskedLayer):
+    '''
+    This layer is used to normalize across a tensor axis.  Normalization is done by finding the minimum value
+    across the specified axis, and then subtracting that value from all values (again, across the spcified axis).
+    '''
+    def __init__(self, axis, **kwargs):
+        self.axis = axis
+        super(SubtractMinimum, self).__init__(**kwargs)
+
+    def build(self, input_shape):
+        # Add the trainable weight variable for the noise parameter.
+        super(SubtractMinimum, self).build(input_shape)
+
+    def get_output_shape_for(self, input_shape): # pylint: disable=no-self-use
```

Comment:
This should be `compute_output_shape`.  Also, can you add `@overrides` tags to all of the methods here that override superclass methods?  That would have caught this issue.